qxf2/qxf2-lambdas
# Test url filter lambda on localstack

Test URL filter lambda on LocalStack:
Test will do following:
- Deploy Lambda along with its dependencies and SQS
- Run test
- Delete Lambda and SQS queue

Setup:
- cd to tests directory
- Start LocalStack using following command:
`docker-compose up -d`
- Check LocalStack status:
`localstack status`

Run Test:
- cd to tests directory
- pytest -s -v test_url_filter_lambda_on_localstack.py

Test ran successfully
![url-filter-lambda-test-run](https://github.com/qxf2/qxf2-lambdas/assets/17157735/99b42035-1e89-43e0-b751-aa2360b07aa5)
